test/e2e/karmadactl_test.go

```diff
@@ -992,3 +993,181 @@ var _ = ginkgo.Describe("Karmadactl describe testing", func() {
 		})
 	})
 })
+
+var _ = ginkgo.Describe("Karmadactl token testing", func() {
+	ginkgo.Context("Test creating a bootstrap token", func() {
+		var (
+			token string
+			err   error
+		)
+
+		ginkgo.AfterEach(func() {
+			cmd := framework.NewKarmadactlCommand(kubeconfig, karmadaContext, karmadactlPath, "", karmadactlTimeout, "token", "delete", token)
+			_, err := cmd.ExecOrDie()
+			gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
+		})
+
+		ginkgo.It("should create a token successfully and validate its format", func() {
+			cmd := framework.NewKarmadactlCommand(kubeconfig, karmadaContext, karmadactlPath, "", karmadactlTimeout, "token", "create")
+			token, err = cmd.ExecOrDie()
+			token = strings.TrimSpace(token)
+			gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
+
+			// Validate the token format.
+			tokenID, tokenSecret := extractTokenIDAndSecret(token)
+			gomega.Expect(validateTokenFormat(tokenID, tokenSecret)).Should(gomega.BeTrue(), "Token format is invalid")
```

Comment:
Can we use the following pattern for matching?
https://github.com/karmada-io/karmada/blob/32924ad5d88ab1e0226f653f70ef03703343d504/vendor/k8s.io/cluster-bootstrap/token/api/types.go#L98-L99

Reply:
Yes, we could use that pattern. However, I think it's more readable to break down the regex like this as in the test case:
```go
func validateTokenFormat(tokenID, tokenSecret string) bool {
    tokenIDPattern := regexp.MustCompile(`^[a-z0-9]{6}$`)
    tokenSecretPattern := regexp.MustCompile(`^[a-z0-9]{16}$`)
    return tokenIDPattern.MatchString(tokenID) && tokenSecretPattern.MatchString(tokenSecret)
}
```

But you are right, consistency is more important, so using this global constant for matching bootstrap tokens is a good idea. 👍

Reply:
Thanks~
Is it necessary to parse the tokenID and tokenSecret first?

Reply:
Do you mean using those in here the `validateTokenFormat` function rather than in the `extractTokenIDAndSecret` function? 

For the `validateTokenFormat` function not necessary to parse them first, I was just starting to learn about the Karmada bootstrap token functionality, so breaking things up made it easier for me to understand.

For the `extractTokenIDAndSecret` function, yes, we need to parse them to test for both token creation and deletion.

Reply:
Ok, thanks for your explanation!